### PR TITLE
Transition _buildDynamicOption code

### PR DESCRIPTION
**Pull request by ghost**
Work in progress pull request for transitioning the code for dynamic series.

### CODE REVIEW

File: appserver/static/visualizations/hman/src/buildDynamicOption/index.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  option = this._parseOption(configOption);
+  if (option == null) {
+    return null;
+  }
+
+  // array with list of comma separated values provided in configXAxisDataIndexBinding
+  var xAxisDataIndex = [];
+  xAxisDataIndex = this._parseIndex(configXAxisDataIndexBinding);
+
+  var seriesDataIndexes = configSeriesDataIndexBinding.split(";").map(Number);
```
Comment (LorackDev) [inline appserver/static/visualizations/hman/src/buildDynamicOption/index.js:17]: seriesDataIndex initialisation is different than in _buildCustomOption

File: appserver/static/visualizations/hman/src/buildDynamicOption/index.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    }
+  }
+  if (configXAxisDataIndexBinding != null) {
+    if (option.hasOwnProperty('xAxis')) {
+      option['xAxis'] = xAxisObjects;
+    } else {
+      option.xAxis = xAxisObjects;
+    }
+  }
+
```
Comment (LorackDev) [inline appserver/static/visualizations/hman/src/buildDynamicOption/index.js:67]: up to this point identical to _buildCustomOption, except where annotated

File: appserver/static/visualizations/hman/src/buildDynamicOption/index.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    for (var i = 1; i < data.fields.length; i++) {
+      // If the series is already specified, skip it
+      if (seriesDataIndexes.indexOf(i) !== -1) continue;
+
+      seriesList.push(createSeries(defaultName + i, defaultType, i, defaultSmooth, defaultConnectNulls, defaultShowSymbol));
+    }
+  }
+
+  option.series = seriesList;
+
```
Comment (LorackDev) [inline appserver/static/visualizations/hman/src/buildDynamicOption/index.js:116]: following error handling up to the end of the file is identical to _buildCustomOption

**Comment by ghost** [comment]
Replaced by https://github.com/hman-78/visualization_toolbox/pull/49

**Pull request closed without merging**